dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js

```diff
@@ -0,0 +1,36 @@
+var messages = {
+    WELCOME_OPEN_LIBERTY: "Welcome to Open Liberty",
+    OPEN_LIBERTY_APP_SERVER: "Open Liberty Application Server",
+    WEBSPHERE_OPEN_LIBERTY_LOGO: "WebSphere Liberty Logo",
```

Comment:
and I hate to be picky but it isn't "WebSphere Open Liberty", just "Open Liberty". Please change the variable names as well as the text.